api/src/feeds/impl/models/latest_dateset_impl.py

```diff
@@ -0,0 +1,48 @@
+from functools import reduce
```

Comment:
Typo in file name name (dat**e**set instead of dataset)

Reply:
Fixed!